migrations/20221220120527-add-transaction-subscription-indexes.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.addIndex('Transactions', ['CollectiveId', 'createdAt', 'type']);
```

Comment:
Should we remove the existing `CollectiveId-type` index in favor of this one?

Reply:
@hdiniz ?

Reply:
`CollectiveId-Type` is one of the most used indexes on the transactions table, I would look at that on a separate issue to see the query impact. 

This new index might not be compatible with `"CollectiveId-type" btree ("CollectiveId", type)` as it will be ordered by the createdAt before type.

```
select indexrelname, idx_scan, pg_size_pretty(pg_relation_size(indexrelid)) as "Index Size" from pg_stat_all_indexes where schemaname = 'public' and relname = 'Transactions' order by idx_scan desc;
                   indexrelname                   | idx_scan  | Index Size
--------------------------------------------------+-----------+------------
 CollectiveId-FromCollectiveId-type               | 751364426 | 61 MB
 transactions__using_gift_card_from_collective_id | 654061807 | 42 MB
 CollectiveId-type                                | 471101054 | 50 MB
```

Reply:
Let's merge like this then.